src/render/SDL_render.c

```diff
@@ -3253,6 +3256,7 @@ bool SDL_SetRenderDrawColorFloat(SDL_Renderer *renderer, float r, float g, float
     renderer->color.g = g;
     renderer->color.b = b;
     renderer->color.a = a;
+
     return true;
```

Comment:
Revert extra newline.
```suggestion
    renderer->color.a = a;
    return true;
```